experimental/packages/opentelemetry-instrumentation/src/instrumentation.ts

```diff
@@ -136,7 +136,7 @@ export abstract class InstrumentationAbstract<T = any>
    * Init method in which plugin should define _modules and patches for
    * methods
```

Comment:
Since this is public now, I'm wondering if we should extend documentation to tell users to use `enable` if they're looking to enable the instrumentation. I could see it being a bit confusing in some cases to have `enable()` and `init()` both public.

Reply:
Good call. I'll update the jsdocs

Reply:
Added that. Lmk what you think